metaswitch/common/alarms_parser.py

```diff
@@ -30,37 +30,41 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 import json
+import alarm_severities
 
-# Valid severity levels - this should be kept in sync with the 
+# valid severity levels - this should be kept in sync with the
```

Comment:
Why have you uncapitalised this? Why actively degrade our English?
Change aaaaaalllll this back.